src/pages/workspaces/workspace/analysis/runtime-utils.js

```diff
@@ -0,0 +1,316 @@
+import _ from 'lodash/fp'
```

Comment:
I think something has gone very wrong with merging in this file. This file no longer exists in the UI, and is in `runtime-utils.ts`. I recommend deleting it on your branch, and ensuring any changes to methods that this PR depends on ends up in the related files. For example, most of the disk-related code now lives in `disk-utils.ts` or `disk-models.ts`